docs/usage.md

```diff
@@ -104,6 +107,28 @@ Sample files that can be used as illustrative references can be found in this re
 | last_name | str | User's last name |
 | email | str | User's email |
 
+## `ib_input_instances.csv`
```

Comment:
Same as above: Can we add these two tables after the docs for `grades.csv` so it is alphabetical?